Puppetfile.lock

```diff
@@ -17,7 +17,11 @@ FORGE
 GIT
   remote: https://github.com/Katello/puppet-candlepin
   ref: master
+<<<<<<< HEAD
```

Comment:
Good catch, although this of little consequence in reality for our release branches given the way we store modules and cherry pick.